### ISSUE TITLE
Abnormal keto acid urine concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6019]
**Preferred term label:** Abnormal keto acid urine concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Abnormal levels of a keto acid in the urine. 

**Parent term (use hpo.jax.org/app)** Organic aciduria HP:0001992

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Elevated urine 3-methyl-2-oxovaleric acid concentration

**Synonyms** Elevated 2-keto-3-methylvaleric acid

**Definition (free text, please give PubMed ID)** Concentration of  3-methyl-2-oxovaleric acid in the urine is above the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal keto acid urine concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Elevated urine alpha-ketoisocaproic acid concentration

**Synonyms** Elevated urine 2-ketoisocaproic acid concentration

**Definition (free text, please give PubMed ID)** Concentration of  alpha-ketoisocaproic acid in the urine is above the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal keto acid urine concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Elevated urine alpha ketoisovaleric acid concentration

**Synonyms** 

**Definition (free text, please give PubMed ID)** Concentration of  alpha ketoisovaleric acid in the urine is above the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal keto acid urine concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added terms
